scripts/vit-watchdog

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+#
+# Watchdog script for vit -- automatically reloads vit when task data changes.
+# Monitors the pending.data file and sends a SIGUSR1 signal to vit to trigger a refresh.
```

Comment:
I mentioned this before, we definitely want to add a note making it clear this script ONLY works with Taskwarrior < 3.0.

Reply:
done.